### DISCUSSION TITLE
Update an app from a shared folder from Mega cloud storage

### DISCUSSION THREAD

**Discussion by summoner001** [ImranR98/Obtainium#1273] [Ideas]
Hi!

Would it be possible for Obtainium to be able to update from a shared folder in the cloud? No login required. Just somehow load this example folder: 

Example shared folder with apk files: https://mega.nz/folder/lc4RlIaS#gdGFUDzYkXhQejA1ry1Vag

It would be easier to share my apk files with my friends.

**Comment by sunnyd24** [comment]
This would be great, I can see some good uses for sharing apks via Mega shared folders.

**Comment by summoner001** [comment]
☹️
